src/main.rs

```diff
@@ -232,6 +232,31 @@ fn print_table<T: std::fmt::Display + Clone>(mut input: Vec<T>, columns: usize,
     table.printstd();
 }
 
+fn dep_wanted(preconditions: &When, settings: &serde_json::Value) -> bool {
```

Comment:
as a follow-on, we might add `any_of` support to the runscripts & renderer as well.